src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/transformer/DocumentTransformer.kt

```diff
@@ -14,16 +14,7 @@ class DocumentTransformer {
 
   private val log = LoggerFactory.getLogger(this::class.java)
 
-  fun transformToApiFiltered(groupedDocuments: GroupedDocuments, convictionId: Long?) =
-    transformToApi(groupedDocuments) { it.convictionId == convictionId.toString() }
-
-  fun transformToApiUnfiltered(groupedDocuments: GroupedDocuments) =
-    transformToApi(groupedDocuments) { true }
-
-  fun transformToApi(
-    groupedDocuments: GroupedDocuments,
-    convictionDocFilter: (ConvictionDocuments) -> Boolean,
```

Comment:
no longer need to filter by convictionId due to the discovery (as mentioned in PR description) that the 'Temporary Accommodation' flow is no longer used so the related business-logic can be stripped been stripped out.